### PR TITLE
Fix UG bugs and modify claims error messages

### LINKED ISSUE #124: [PE-D] Insufficient instructions about input format in UG
Expected behaviour: Readers can learn about restrictions on inputs, like email format, tag format, "names need to be unique", "status can only be pending or completed".

Actual behaviour: There are only error messages to indicate wrong input format. The error message for email format is really long without indicating where *exactly* my email address goes out of the format.

Suggestion: a better approach could be moving the format instructions to UG and providing some examples. Learning how to use the app from *UG* could provide a better user experience compared to learning from *error messages*.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: TTraveller7/ped#3

### LINKED ISSUE #125: [PE-D] Wrong Screenshot used for find command
1. The screenshots used does not match the current app.

2. `find` command seems to be able to locate the person's appointment and claims. However, this behaviour is not specified in UG.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: TTraveller7/ped#4

### LINKED ISSUE #127: [PE-D] Same tag names cannot be repeatedly added to the same person
`add n/n5 t/w t/w t/w`

Expected behaviour: 3 tags named "w" is added

Actual behaviour: only 1 is added

Suggestion: Maybe this could be specified in UG

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: TTraveller7/ped#6

### LINKED ISSUE #128: [PE-D] UG of claim command is unclear
1. The feature title says it is for "Adding a new claim". However, the *same* command is also used to edit and remove claims. These uses are only revealed in examples.

2. The `TITLE` field is case-sensitive, yet this is not specified in UG or error message.

3. The `INDEX` and `TITLE` field seems to be compulsory fields in any use, while the other two are optional. However, this is not specified correctly by either UG or error message.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: TTraveller7/ped#7

### LINKED ISSUE #134: [PE-D] UG has some references to addressbook
The UG still refers to the app ass Address Book, which can lead to a bit of confusion. 
 
It might be better to update it to refer to your app instead.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: clyveycui/ped#3

### LINKED ISSUE #136: [PE-D] UG Typo for revenue command
There is a mismatch of instructions between the section 'Command Summary' and 'Adding Revenue: revenue'

This is the command format under Adding Revenue: revenue:

This is the command format under command summary: 

tldr: typo of `revenue r/-100.11` as `revenue -100.11`

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: ruoyann/ped#1

### LINKED ISSUE #137: [PE-D] UG Inaccuracy in Quick Start
We are instructed to open the .jar files by running the -jar command instead of double clicking. Your team's quick start suggested users to double click instead.

Where I got the screengrab from: https://nus-cs2103-ay2122s1.github.io/website/admin/tp-pe.html

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: ruoyann/ped#2

### LINKED ISSUE #139: [PE-D] Parameter constraints for tags are not clear
Tried to execute the following command to find out that tags should be alphanumeric and disallow hyphens and whitespace.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: NicolasCwy/ped#3

### LINKED ISSUE #141: [PE-D] Edit command behaves differently than described
In the UG, it is not stated that you can update the note in the command format, which is different from the in app usage message.

Perhaps it would be better to mention that inside the UG.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: clyveycui/ped#4

### LINKED ISSUE #147: [PE-D] Parameter Constraints for claim status not clear
The GUI provides a helpful error saying that the claim status can only be "pending" or "complete" but this can be included in your UG as well! Your UG currently says that `STATUS is a string describing the status of the insurance claim` which is slightly misleading since there is validation

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: NicolasCwy/ped#6

### LINKED ISSUE #150: [PE-D] Claim command has different behavior than as described in the usage message
The usage message suggests that the status, title, and descriptions are optional parameters, but there are actually required in the command.

It is also said that "Claims with the same name will be overwritten", but when adding a claim with the same name but with a different person, it ends up creating a new claim instead.

Perhaps it should be updated.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: clyveycui/ped#6

### LINKED ISSUE #151: [PE-D] Claim command format is inaccurate
Name prefix is t/ in the UG when it should n/

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: NicolasCwy/ped#7

### LINKED ISSUE #152: [PE-D] `claim` Command - User guide prefix mismatch from command error message
* Prefix for title is `n/` not `t/` stated in the user guide. 

* The command error message also uses square brackets, which indicate optional fields according to the Notes under the `Features` header of the user guide. Could remove the square brackets since the fields are not optional.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: tanqiya/ped#3

### LINKED ISSUE #156: [PE-D] `note` command - prefix not updated
Should be `no/` instead of `n/`. Could also refer to the client list rather than address book since it is named that way in the GUI.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: tanqiya/ped#4

### LINKED ISSUE #157: [PE-D] Note command incorrect format in UG
The note command only accepts the prefix no/, from the useful error thrown in the GUI

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: NicolasCwy/ped#9

### LINKED ISSUE #163: [PE-D] UG differs from the in app usage message for Note command
The parameter prefix used in the UG is n/, not no/

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: clyveycui/ped#9

### LINKED ISSUE #168: [PE-D] Command summary table errors
Here are the few that I spotted:

Add command missing no/NOTE

Claim wrong prefix for title

Edit missing no/NOTE

Note and Revenue

Might contain duplicates bug reports

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: clyveycui/ped#12

### PR DESCRIPTION
**Pull request by JoelChanZhiYang**
Resolves #168
Resolves #147
Resolves #163
Resolves #157
Resolves #156
Resolves #152
Resolves #151
Resolves #150
Resolves #147
Resolves #128
Resolves #127
Resolves #125
Resolves #124
Resolves #141
Resolves #139
Resolves #137
Resolves #136
Resolves #134

### CODE REVIEW

**Comment by JoelChanZhiYang** [comment]
Resolves #168
Resolves #147
Resolves #163
Resolves #157
Resolves #156
Resolves #152
Resolves #151
Resolves #150
Resolves #147
Resolves #128
Resolves #127
Resolves #125
Resolves #124
Resolves #141
Resolves #139
Resolves #137
Resolves #136
Resolves #134

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,34 +18,49 @@ Insurance Pal is a client management software specially designed to help manage
...
 
-## Features
-
-<div markdown="block" class="alert alert-info">
+## Command Summary
+
+Action | Parameters| Description
+--------|------------------|-------
+`add n/NAME p/PHONE e/EMAIL a/ADDRESS [no/NOTE] [t/TAG]… [i/INSURANCE]…` |`NAME`: Name of the client <br> `PHONE`: Phone number of the client <br> `EMAIL`: Email of the client <br> `ADDRESS`: Address of the client <br> `NOTE`: Note about the client<br>`TAG`: Tags of the client <br>`INSURANCES`: life/health/general <img>| Adds a client to InsurancePal
+`edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/REVENUE] [a/ADDRESS] [n/NOTE] [t/TAG]… [i/INSURANCE]…` | `INDEX`: Index of the client <br> `NAME`: Name of the client <br> `PHONE`: Phone number of the client <br> `EMAIL`: Email of the client <br>  `REVENUE`: Revenue of the client <br>`ADDRESS`: Address of the client <br> `NOTE`:Note about the client  <br>`TAG`: Tags of the client <br>`INSURANCES`: life/health/general | Edits a client information
```
Comment (xianlinc) [inline docs/UserGuide.md:34]: edit note should be no/ instead of n/

File: docs/UserGuide.md [hunk 2]
```diff
@@ -54,172 +69,313 @@ Insurance Pal is a client management software specially designed to help manage
...
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a client : `edit`
 
-Edits an existing person in the address book.
+You can edit an existing client using the `edit` command
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/REVENUE] [a/ADDRESS] [t/TAG]… [i/INSURANCE]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/REVENUE] [a/ADDRESS] [n/NOTE] [t/TAG]… [i/INSURANCE]…`
```
Comment (xianlinc) [inline docs/UserGuide.md:147]: note prefix should be no/

**Review by xianlinc — CHANGES_REQUESTED** [review]
Other than these two changes, the rest of the file LGTM
